README.md

```diff
@@ -1 +1,5 @@
-# gatk-evaluation
\ No newline at end of file
+# gatk-evaluation
+
+## Docker Repo 
+
+`us.gcr.io/broad-dsde-methods/gatk-evaluation/`
```

Comment:
I should probably push my evaluation images there too. Do I need to have any permissions?

Reply:
@asmirnov239 You should already have access.  Tell me if not.